BTrees/tests/test_OOBTree.py

```diff
@@ -126,6 +126,12 @@ class C(object):
 
         self.assertRaises(TypeError, lambda : t.__setitem__(C(), 1))
 
+        try:
+            t[C()] = 1
+            self.fail("Should raise TypeError")
+        except TypeError as e:
+            self.assertEqual(e.args[0], "Object has default comparison")
+
```

Comment:
I always forget that it can be used as a context manager. Thanks, will do!
(I write most of my tests with PyHamcrest, where you would write
```python
assert_that( calling(t.__setitem__).with_args(C(), 1),
                     raises(TypeError, "Object has default comparison"))
```
)